Tests/MapboxMapsTests/Foundation/Camera/MapboxMapsCameraTests.swift

```diff
@@ -368,4 +368,20 @@ class CameraManagerTests: XCTestCase {
         XCTAssertTrue(cameraManager.cameraAnimators.contains(thirdAnimator))
         XCTAssertTrue(cameraManager.cameraAnimators.contains(fourthAnimator))
     }
+
+    func testCancelTransitions() {
+        XCTAssertTrue(cameraManager.cameraAnimators.count == 0)
+        let runningAnimator = cameraManager.makeRunningCameraAnimator(duration: 100, delay: 2) { [weak self] in
+            let coordinate2 = CLLocationCoordinate2D(latitude: 0, longitude: 0)
+            self?.cameraManager.setCamera(centerCoordinate: coordinate2, zoom: 16)
+        }
+        XCTAssertTrue(cameraManager.cameraAnimators.count != 0)
+        XCTAssertTrue(cameraManager.cameraAnimators.contains(runningAnimator))
+        
+        cameraManager.cancelTransitions()
+
+        DispatchQueue.main.asyncAfter(deadline: .now() + 0.1) { [weak self] in
+            XCTAssertTrue(self?.cameraManager.cameraAnimators.count == 0)
+        }
+    }
```

Comment:
Hmm so this isn't a valid test -- the `cameraAnimators.count` is becoming `0` because the `runningAnimator` is falling out of scope.